src/Models/Command.tsx

```diff
@@ -0,0 +1,10 @@
+export type CommandCallback<> = () => void
+
+export type Command = {
+    icon?: string
+    id: number
+    name: string
+    tooltip: string
+    // TODO: Add a possible shortcut to commands
+    callback: CommandCallback
+};
```

Comment:
```suggestion
};

```
Add newline at the end of file